vendor/github.com/prometheus/common/config/http_config.go

```diff
@@ -42,6 +39,7 @@ import (
 // DefaultHTTPClientConfig is the default HTTP client configuration.
 var DefaultHTTPClientConfig = HTTPClientConfig{
 	FollowRedirects: true,
+	EnableHTTP2:     true,
```

Comment:
This replaces the previous env var `PROMETHEUS_COMMON_DISABLE_HTTP2` below. But afaik we're not using this.